WEBSITE_CONTRIBUTING.md

```diff
@@ -11,6 +11,77 @@ Contributing to the Podman Desktop website is a valuable way to support the proj
 
 The below information outlines details on how to both contribute as well as our documentation "templates" and criteria.
 
+## Getting Started
+
+### Install the project dependencies
+
+```shell-session
+$ pnpm install
+```
+
+### Local Development of the website / documentation
+
+```shell-session
+$ pnpm start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+You may also run:
+
+```sh
+pnpm website:dev
+```
+
+Which will automatically open your browser to `localhost:3000`.
+
+### Build
+
+```shell-session
+$ pnpm build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+You may also build a "production-like" environment by using:
+
+```sh
+pnpm website:prod
+```
+
+### Deployment
+
+Using SSH:
+
+```shell-session
+$ USE_SSH=true pnpm deploy
+```
+
+Not using SSH:
+
+```shell-session
+$ GIT_USER=<Your GitHub username> pnpm deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
```

Comment:
Since we have already mentioned `The website is built using [docusaurus](https://docusaurus.io/) and published to [podman-desktop.io](https://podman-desktop.io/) on each commit.`, how this piece of information would be useful for a website contributor? They can build the website and see its preview even without running these commands.

A contributor might see it as an additional step to do, which is not at all required.